actors/runtime/src/actor_error.rs

```diff
@@ -104,4 +139,40 @@ macro_rules! actor_error {
     ( $code:ident, $msg:literal $(, $ex:expr)+ ) => {
         $crate::actor_error!($code; $msg $(, $ex)*)
     };
+
+    ($code:ident, $fmt:expr, $($arg:tt)*) => {
+        $crate::actor_error!($code, format!($fmt, $($arg)*))
+    };
+}
+
+#[macro_export]
+macro_rules! ensure {
+    ($cond:expr, $code:ident, $msg:literal $(,)?) => {
+        if !$cond {
+            return Err($crate::actor_error!($code, $msg).into());
```

Comment:
Curious: why use `into` rather than just construct the `Abort` explicitly here?

Reply:
Because it is not possible to construct an `Abort`, only be triggering the `into` implementation. This does not necessarily convert into an `Abort`, it could just convert into itself or other Error types depending on where it is used.